### PR TITLE
Add tutorial for a dockerized cs:go dedicated server

### PR DESCRIPTION
**Pull request by lukasboettcher**
I have read and understood the Contributor's Certificate of Origin at the end of the template and I hereby certify that I meet the contribution criteria described in it.

Signed-off-by: Lukas Böttcher <contact@btchr.de>

### CODE REVIEW

File: community-tutorials/dockerized-csgo-gameserver/01-en.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+This requires that you previously created a directory named ```/backup``` on your server.
+This is where the copied files from the container will end up.
+# Conclusion
+Now you should have a running CS:GO dedicated server that keeps itself up to date.
+All data is stored in the ```csgo_data``` docker volume, which persists the data even of you remove the container.
+
+You can now connect to your server at ```steam://connect/<example.com>/<sv_password>``` or via the community server browser in game.
+
+All steps are essentially the same for other source games from Valve, such as Garrys Mod or CSS.
+
```
Comment (d-schaffer) [inline community-tutorials/dockerized-csgo-gameserver/01-en.md:322]: It is difficult to distinguish which commands I have to execute locally and which I have to execute on the server.
Please divide it better (locally or on the server).
Above all, it is primarily important how I set it up on the Netcup product. 
The local variant is rather secondary.

File: community-tutorials/dockerized-csgo-gameserver/01-en.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+docker run -d --rm -it -v csgo_data:/data -v /backup:/backup ubuntu
+```
+then simply copy all the files you want to backup from ```/data``` to ```/backup``` inside the container.
+This requires that you previously created a directory named ```/backup``` on your server.
+This is where the copied files from the container will end up.
+# Conclusion
+Now you should have a running CS:GO dedicated server that keeps itself up to date.
+All data is stored in the ```csgo_data``` docker volume, which persists the data even of you remove the container.
+
+You can now connect to your server at ```steam://connect/<example.com>/<sv_password>``` or via the community server browser in game.
```
Comment (d-schaffer) [inline community-tutorials/dockerized-csgo-gameserver/01-en.md:319]: Server is not responding.
-> probably resulting from the misconfigured CSGO container

**Review by d-schaffer — COMMENTED** [review]
Please revise the tutorial again

File: community-tutorials/dockerized-csgo-gameserver/01-en.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+to start the container with default ports. If you plan on running multiple containers or prefer different ports, you need to adjust the port mappings accordingly with the ```-p``` argument.
+
+If your created all files directly on your server, simply omit the ```-H "ssh://user@example.com"``` argument.
+
+Now the container should be started and will begin downloading the CS:GO dedicated server.
+This can take some minutes, since there are about 20GB of server files to be downloaded.
+
+You can see the logs by running:
+```bash
+docker -H "ssh://user@example.com" logs -f csgo
```
[the remark this answers is not in the document]
Comment (lukasboettcher) [reply community-tutorials/dockerized-csgo-gameserver/01-en.md:281]: That means that there was not enough free space left on the disk, I added this as a requirement.

File: community-tutorials/dockerized-csgo-gameserver/01-en.md [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+sv_log_onefile 1
+sv_hibernate_when_empty 1 
+sv_hibernate_ms 5 
+sv_maxrate "0" 
+sv_minrate "128000" 
+sv_minupdaterate "128"
+sv_mincmdrate "128" 
+sv_pure 1 
+sv_pure_kick_clients 1 
+hostname ""                 // change this
```
[the remark this answers is not in the document]
Comment (lukasboettcher) [reply community-tutorials/dockerized-csgo-gameserver/01-en.md]: they can all be left blank
I added more detailed comments for these settings

File: community-tutorials/dockerized-csgo-gameserver/01-en.md [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+---
+
+# Introduction
+In this tutorial you will learn how to run gameservers in a docker environment.
+Specifically a Counter-Strike: Global Offensive server that can automatically update itself.
+
+# Requirements
+- A server, ideally hosted by netcup :), that can handle running gameserver instances. - At least an RS 1000 is recommended for hosting multiple gameservers.
+- A working docker installation (see docker installation tutorial) on the server (and optionally on your local computer)
+- Root access to your server via ssh (ideally pubkey based)
```
[the remark this answers is not in the document]
Comment (lukasboettcher) [reply community-tutorials/dockerized-csgo-gameserver/01-en.md:24]: owning the game is not required for running the server, but a steam account is.
added that as another requirement.

File: community-tutorials/dockerized-csgo-gameserver/01-en.md (hunk 2 above)
Comment (lukasboettcher) [reply community-tutorials/dockerized-csgo-gameserver/01-en.md]: yes, server was probably not downloaded fully for lack of free space on the server.

**Comment by netcupClaudiaM** [comment]
Hi lukasboettcher, thank you for contributing! It's been a while since you opened this PR and it probably is an open secret we're currently overwhelmed with reviewing all the awesome tutorials we got. Please be patient - we're about to publish tutorials this and next week. Thank you!

**Pull request merged**